src/pathfora.js

```diff
@@ -1283,21 +1258,21 @@
               ct;
 
           if (val) {
-            val = val.split("|");
+            val = val.split('|');
             // NOTE Retain support for cookies with comma - can remove on 5/2/2016
```

Comment:
lets make another ticket to remove the cookie comma retention stuff but keep it separate / post the linting one.